addons/sourcemod/scripting/shop/functions.sp

```diff
@@ -284,6 +283,45 @@ public int Functions_Menu_Handler(Menu menu, MenuAction action, int param1, int
 	}
 }
 
+public void Function_ConfirmLuckMenu(int client)
+{
+	char buffer[256];
+	Menu menu = new Menu(Menu_ConfirmTryLuck);
+
+	FormatEx(buffer, sizeof(buffer), "%T \n", "confirm_luck", client, g_hLuckCredits.IntValue);
+	menu.SetTitle(buffer);
+	FormatEx(buffer, sizeof(buffer), "%T", "Yes", client);
+	menu.AddItem("yes", buffer);
+	FormatEx(buffer, sizeof(buffer), "%T", "No", client);
+	menu.AddItem("no", buffer);
+
+	menu.ExitBackButton = false;
+	menu.ExitButton = false;
+	menu.Display(client, MENU_TIME_FOREVER);
+}
+
+public int Menu_ConfirmTryLuck(Menu menu, MenuAction action, int param1, int param2)
+{
+	switch (action)
+	{
+		case MenuAction_Select:
+		{
+			char info[16];
+			menu.GetItem(param2, info, sizeof(info));
+		
+			if(StrEqual(info, "yes"))
```

Comment:
instead of string comparison, we could use first symbol
```suggestion
			if (info[0] == "y")
```

Reply:
`e:\Linux\Shop-Core\addons\sourcemod\scripting\shop/functions.sp(316) : error 033: array must be indexed (variable "-unknown-")`

Reply:
Changed to `if (info[0] == 'y')`